subsys/debug/coredump/coredump_backend_flash_partition.c

```diff
@@ -376,22 +375,15 @@ static void coredump_flash_backend_end(void)
 	hdr.error = backend_ctx.error;
 	hdr.flags = 0;
 
-	flash_dev = flash_area_get_device(backend_ctx.flash_area);
-
-	/* Need to re-init context to write at beginning of flash */
-	ret = stream_flash_init(&backend_ctx.stream_ctx, flash_dev,
-				stream_flash_buf,
-				sizeof(stream_flash_buf),
-				backend_ctx.flash_area->fa_off,
-				backend_ctx.flash_area->fa_size, NULL);
-	if (ret == 0) {
-		ret = stream_flash_buffered_write(&backend_ctx.stream_ctx,
-						  (void *)&hdr, sizeof(hdr),
-						  true);
-		if (ret != 0) {
-			LOG_ERR("Cannot write coredump header!");
-			backend_ctx.error = ret;
-		}
+	/*
+	 * Write header without using a stream context
+	 * If CONFIG_STREAM_FLASH_ERASE is set, this would clear an entire erase page,
```

Comment:
This comment should be removed, it has no meaning outside the PR.

Reply:
I added the comment so nobody in the future would wonder why a stream context isn't used here.

Reply:
I know, but "this" does not refer to anything outside the PR.